Makefile

```diff
@@ -0,0 +1,25 @@
+.PHONY : install test serve lint
+
+POETRY := $(shell command -v poetry 2> /dev/null)
+MKDOCS := $(shell command -v mkdocs 2> /dev/null)
+
+all: install test serve
+
+install:
+	${POETRY} install
+
+test:
+	${POETRY} run pytest
+
+
+serve:
```

Comment:
Maybe this should be called `serve-docs`? Otherwise it looks like you would be serving the project itself (at least for me)

Reply:
:smile: Actually it was, sure is a misconception.